### PR TITLE
lsdk 1906 patches

### PR DESCRIPTION
**Pull request by guochunrong**

### CODE REVIEW

File: recipes-extended/dpdk/dpdk.inc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+do_configure () {
+	#############################################################
+	### default value for prefix is "usr", unsetting it, so it
+	### will not be concatenated in ${RTE_TARGET}/Makefile
+	### which will cause compilation failure
+	#############################################################
+	unset prefix
+
+	make O=$RTE_TARGET T=$RTE_TARGET config
```
Comment (otavio) [inline recipes-extended/dpdk/dpdk.inc]: Use `oe_runmake`

File: recipes-extended/dpdk/dpdk.inc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+export ICP_LIB_ROOT= "${PKG_CONFIG_SYSROOT_DIR}/usr/lib"
+export RTE_KERNELDIR = "${STAGING_KERNEL_DIR}"
+export RTE_KERNELDIR_OUT = "${STAGING_KERNEL_BUILDDIR}"
+export INSTALL_PATH = "${prefix}/share/dpdk"
+export RTE_OUTPUT = "${S}/${RTE_TARGET}"
+export ETHTOOL_LIB_PATH = "${S}/examples/ethtool/lib/${RTE_TARGET}/"
+export SYSROOTPATH = "--sysroot=${STAGING_DIR_HOST}"
+export ICP_LAC_API_DIR = "${STAGING_DIR_TARGET}${includedir}/lac"
+export OPENSSL_PATH = "${STAGING_DIR_HOST}"
+export EXAMPLES_BUILD_DIR = "${RTE_TARGET}"
```
Comment (otavio) [inline recipes-extended/dpdk/dpdk.inc]: We have standard variables for this so please set this on EXTRA_OEMAKE and in other fields refer to our standard variables such as  `datadir`, and so on.

File: recipes-extended/dpdk/dpdk.inc [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+"
+FILES_${PN}-dev += " \
+	${INSTALL_PATH}/${RTE_TARGET}/.config \
+	${includedir} \
+	${includedir}/${ARCHDIR} \
+	${includedir}/exec-env \
+	${INSTALL_PATH}/buildtools/ \
+	${INSTALL_PATH}/${RTE_TARGET}/include \
+	${INSTALL_PATH}/${RTE_TARGET}/lib \
+	${INSTALL_PATH}/mk \
```
Comment (otavio) [inline recipes-extended/dpdk/dpdk.inc]: INSTALL_PATH is not OE known so please use our standard variables.

**Comment by guochunrong** [comment]
new patches has been submitted

**Pull request closed without merging**